gateway_tests/conftest.py

```diff
@@ -434,3 +434,116 @@ def compare_structures(gw_struct, ioc_struct) -> str:
                 f"Element '{key}' : GW has '{gateway_value}', IOC has '{ioc_value}'"
             )
     return "\n\t".join(differences)
+
+
+@contextlib.contextmanager
+def ca_subscription(
+    pvname: str,
+    callback: PyepicsCallback,
+    mask: int = epics.dbr.DBE_VALUE,
+    form: str = "time",
+    count: int = 0,
+    timeout: float = 0.5,
+) -> ContextManager[int]:
```

Comment:
For some reason I've been indenting these close parenthesis/return hint lines, but now that I see it dedented I realize that this is a better way to do it because it separates you from the indent in the function body.

Reply:
I noticed this in your code for a while, but both are technically valid so I didn't want to say "_my way is right_". With all the verbose type hints `black` tends to put things as I have here, and that kinda stuck with me.

Reply:
My thought process was basically "Well, it can't be in line with the code, so it goes one further!", I never considered that I could use the closing parenthesis as the reference line.